public_html/wp-content/mu-plugins/blocks/source/block/organizers/block-controls.js

```diff
@@ -7,49 +7,52 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 const { Button, Placeholder } = wp.components;
+const { Component }           = wp.element;
 const { __ }                  = wp.i18n;
 
 /**
  * Internal dependencies
  */
-import { BlockControls, PlaceholderSpecificMode } from '../shared/block-controls';
-import OrganizersBlockContent                     from './block-content';
-import OrganizersSelect                           from './organizers-select';
-import { LABEL }                                  from './index';
+import { PlaceholderSpecificMode } from '../../component/block-controls';
+import { getOptionLabel }          from '../../component/item-select';
+import { BlockContent }            from './block-content';
+import { Select }                  from './select';
+import { LABEL }                   from './index';
```

Comment:
Generic names here looks way cleaner, but it could cause problem down the line if need to grep for stuff, or if we have conflict with another library. Although I don't feel strongly about this so feel free to keep however :) .

Reply:
That said, perhaps we can think about renaming `Select` to something else. Its way to generic IMO.

Reply:
Fair point. What about `ContentSelect`?

Reply:
Renamed to `ContentSelect`: c89a50f75be17fca4e69e78f98867e03ccd523c5